Content/Part 3/14-a-new-logo.md

```diff
@@ -68,7 +68,7 @@ More designs were posted to the group:
 *Further iterations on the logo design from Jason Santa Maria.*
 
 The members of the mailing list didn’t seem to agree on WordPress’ aesthetic. On one hand, there were people who felt that the logo should represent warmth and community, and on the other hand, something classic and elegant. To reach a consensus, discussions happened offline. Khaled <a href="http://lists.wordpress.org/pipermail/wp-design/2005-April/000192.html">reported back</a>:
-<blockquote>WordPress is meant to be the Jaguar or Aston Martin of Blogging tools. [...] that line sets the stage for what the design of the branding should be. Elegance, polished, and impecably [sic] designed is where we should be aiming.</blockquote>
+<blockquote>WordPress is meant to be the Jaguar or Aston Martin of Blogging tools. [...] that line sets the stage for what the design of the branding should be. Elegance, polished, and impeccably [sic] designed is where we should be aiming.</blockquote>
```

Comment:
This is a case where the text in question is a direct quote copied from its source. The use of "[[sic](https://www.merriam-webster.com/dictionary/sic)]" was added in the book's copy to indicate that the misspelling is known but being left as-is to mirror the original.

Reply:
Understood, wasn't aware of the meaning of [sic], thanks for explaining!